New version: IsotopicCalc v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: IsotopicCalc
- Repository: https://github.com/slowbrain/IsotopicCalc.jl
- Created by: @slowbrain
- Version: v0.3.0
- Commit: a3c2b71408c474008b1884f7d32288fde88d050c
- Reviewed by: @slowbrain
- Reference: https://github.com/slowbrain/IsotopicCalc.jl/issues/14#issuecomment-3567430108
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

This is a breaking release (0.2.x → 0.3.0) with the following changes:

1. **Return Type Change**: `convolve()` and `group_by_resolution()` now return `Vector{Tuple{Float64, Float64}}` instead of `Vector{Pair{Float64, Float64}}` for type consistency with declared return types and improved Julia version compatibility.

2. **Exported Compound Struct**: The `Compound` struct is now publicly exported. If you have your own `Compound` type, use qualified imports to avoid naming conflicts.

See CHANGELOG.md for full migration guide and details.

[noblock]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359893e0a7d8482cc4fa0baae56e33982ce032feaf5e24f4a071741367590c8625093070d76008527024bc4bdda81e572004e63db34ecf8e6edf93cc53fc6d8221a4283a1ae323bfabe9d66b1adaf0c4f75761dd786da047709411bc25e36b0f8a489604c0b92eba14176e3cf5910ef96feb1cf23b21ae3de52f07eb2549c0b08e0b -->

slowbrain commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

This is a breaking release (0.2.x → 0.3.0) with the following changes:

1. **Return Type Change**: `convolve()` and `group_by_resolution()` now return `Vector{Tuple{Float64, Float64}}` instead of `Vector{Pair{Float64, Float64}}` for type consistency with declared return types and improved Julia version compatibility.

2. **Exported Compound Struct**: The `Compound` struct is now publicly exported. If you have your own `Compound` type, use qualified imports to avoid naming conflicts.

See CHANGELOG.md for full migration guide and details.

JuliaRegistrator committed:
New version: IsotopicCalc v0.3.0

UUID: 5554e927-0bdd-4e37-b621-d0b49d571e7b
Repo: https://github.com/slowbrain/IsotopicCalc.jl.git
Tree: 758082fa41e51b1ce1954ae66e7a4d8a948ea8eb

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
